src/game/UIImageTooltipDlg.cpp

```diff
@@ -903,7 +903,7 @@ int CUIImageTooltipDlg::CalcTooltipStringNumAndWrite(__IconItemSkill * spItem, b
             } else {
                 m_pStr[iIndex]->SetColor(m_CRed);
             }
-            sprintf(szBuff, szStr.c_str(), spItem->pItemBasic->byNeedDexterity + spItem->pItemExt->siNeedDexterity);
+            sprintf(szBuff, szStr.c_str(), spItem->pItemBasic->byNeedDexterity + spItem->pItemExt->siNeedDexterity, "");
             m_pstdstr[iIndex] = szBuff;
             iIndex++;
         }
```

Comment:
Checking the binary disassembled code in `CUIImageTooltipDlg::CalcTooltipStringNumAndWrite`:
```c++
      ::_LoadStringFromResource(4538, &szStr);  // Required Dexterity : %d %s
      if ( CUIImageTooltipDlg::SetTooltipTextColor(
             v111[26],
             (unsigned __int8)spItem->pItemBasic->byNeedDexterity + spItem->pItemExt->siNeedDexterity) )
      {
        CN3UIString::SetColor((_DWORD *)this->m_pStr[iIndex], this->m_CWhite);
      }
      else
      {
        CN3UIString::SetColor((_DWORD *)this->m_pStr[iIndex], this->m_CRed);
      }
      v126 = spItem->pItemExt;
      v127 = v126->byMagicOrRare;
      if ( v127 == 4 || v127 == 5 )
      {
        if ( v126->siNeedStrength < 0
          || v126->siNeedStamina < 0
          || v126->siNeedDexterity < 0
          || SLOWORD(v126->siNeedInteli) < 0
          || v126->siNeedInteli < 0 )
        {
          std::string::ctor(v271, &v264);
          LOBYTE(v281) = 13;
          ::_LoadStringFromResource(4554, v271);// (Reduce)
          szReduce = exception::what((exception *)v271);
          iRequiredDex_1 = (unsigned __int8)spItem->pItemBasic->byNeedDexterity + spItem->pItemExt->siNeedDexterity;
          szFmtStr = exception::what((exception *)&szStr);
          sprintf(szBuff, szFmtStr, iRequiredDex_1, szReduce);
          LOBYTE(v281) = 1;
          v129 = v271;
        }
        else
        {
          str::string::equalOperator(v270, &EmptyString, &v264);
          LOBYTE(v281) = 14;
          szEmptyStr = exception::what((exception *)v270);
          iRequiredDex = (unsigned __int8)spItem->pItemBasic->byNeedDexterity + spItem->pItemExt->siNeedDexterity;
          szFmtStr_1 = exception::what((exception *)&szStr);
          sprintf(szBuff, szFmtStr_1, iRequiredDex, szEmptyStr);
          LOBYTE(v281) = 1;
          v129 = v270;
        }
```

We can see that there are some checks whether to add the string `(Reduce)` after level or empty string as you did here. My suggestion here is to do one of the following options:
1. Set all instances we saw in Texts.tbl to empty string **and** a `TODO` comment that `(Reduced)` needs to be implemented. Optionally; no comment and creating a ticket that references the code.
2. Implementing those checks here as per official

I'm giving the options here so you can decide whether you prefer a separate PR for this or reworking the same PR. I'm also open to thoughts if you have any other suggestions.

Reply:
I'll resolve this comment, since the 1st fix has been applied in this commit: https://github.com/ko4life-net/ko/pull/175/commits/6e99e47dfda368f63a601bf55b8101c54ef45bb7

and ticket has been created: https://github.com/ko4life-net/ko/issues/177

Thank you!